frontend/src/views/activity/Activity.vue

```diff
@@ -221,6 +224,14 @@ export default {
       return this.activity.contentNodes()
     }
   },
+
+  // reload data every time user navigates to Activity view
+  async mounted () {
```

Comment:
Hier solltest du zuerst noch `loading` auf `true` setzen oder? Oder werden die data properties neu erstellt wenn dieselbe Komponente erneut gemountet wird? Oder möchtest du absichtlich den letzten bekannten Stand anzeigen? Dann wäre aber allenfalls ein read-only Modus angebracht...

Reply:
`data` wird tatsächlich jedes Mal neu geladen.

Die Idee wäre eigentlich gewesen, dass ich ich den Skeleton loader nur das erste Mal sehe, wenn ich eine Activity neu anschaue. Danach (zweiter Besuch) würden die Daten zwar reloaded (um allfällige Änderungen von anderen Users zu sehen), der skeleton loader muss aber nicht zwingend nochmals angezeigt werden.

Das hatte aber einen Überlegungsfehler drin: Erstens wird eben `loading=true` jedesmal gesetzt und zweitens habe ich unten ja das `await $reload()` drin, was ich bewusst haben will, damit die Daten beim ersten Mal eben nicht zu früh gerendert werden.

Ich füge mal jetzt noch ein `this.loading = true` ein. Kann man dann später immer nochmals optimieren.